src/nimlsp.nim

```diff
@@ -25,27 +27,33 @@ discard existsOrCreateDir(storage)
 infoLog("Version: ", version)
 infoLog("explicitSourcePath: ", explicitSourcePath)
 for i in 1..paramCount():
-  infoLog("Argument " & $i & ": " & paramStr(i))
+  infoLog("Argument ", i, ": ", paramStr(i))
 
 var
   gotShutdown = false
   initialized = false
-  projectFiles = initTable[string, tuple[nimsuggest: NimSuggest, openFiles: OrderedSet[string]]]()
-  openFiles = initTable[string, tuple[projectFile: string, fingerTable: seq[seq[tuple[u16pos, offset: int]]]]]()
+  projectFiles = initTable[string,
```

Comment:
As I said I try to stick with 80 when it makes sense, but mostly limit myself to 128. Breaking up stuff like this doesn't make sense, at least not in this way. Cleaning this up would mean creating named objects instead of those tuples. But I'd be fine with just going back to the oneliners.